tools/explorer/lib/tests/ontology_wrapper_test.py

```diff
@@ -0,0 +1,106 @@
+"""Testing module for ontology.py"""
```

Comment:
please add `.` at the end of the comments. The internal linter flags those.

Reply:
add point at the end. rename the necessary to `ontology_wrapper` ontology is a leftover from the previous code.